tests/page-objects/default/common/common.wdio.page.js

```diff
@@ -18,6 +18,7 @@ const taskTab = () => $('#tasks-tab');
 const getReportsButtonLabel = () => $('#reports-tab .button-label');
 const getMessagesButtonLabel = () => $('#messages-tab .button-label');
 const getTasksButtonLabel = () => $('#tasks-tab .button-label');
+const getAllButtonLabels = async () => await $$('.button-label');
```

Comment:
The `.button-label` is quite common, so better to make the selector specific 
```suggestion
const getAllButtonLabels = async () => await $$('.header .tabs .button-label');
```